cloudant-client/src/main/java/com/cloudant/client/api/query/QueryBuilder.java

```diff
@@ -24,36 +24,144 @@
     private Sort[] sort;
     private Long limit;
     private Long skip;
+    private String bookmark;
+    private boolean update = true;
+    private boolean stable;
+    private boolean executionStats;
+    private String[] useIndex;
 
+    /**
+     * Construct a {@code QueryBuilder} object for use with {@link com.cloudant.client.api.Database#query(String, Class)}
+     * @param selector JSON object describing criteria used to select documents.
+     * @see <a
+     * href="https://console.bluemix.net/docs/services/Cloudant/api/cloudant_query.html#selector-syntax"
+     * target="_blank">selector syntax</a>
+     * @see com.cloudant.client.api.Database#query(String, Class)
+     */
     public QueryBuilder(Selector selector) {
         this.selector = selector;
     }
 
+    /**
+     * Set the fields option for the query builder.
+     * @param fields List specifying which fields of each object should be returned. If it is
```

Comment:
Is "object" correct in this paragraph (twice) vs "document"?

Reply:
This was lifted from the couch docs but could probably do with editing

Reply:
Fixed in 24b2bd8